R/mmrm-package.R

```diff
@@ -16,5 +16,6 @@
 #' @importFrom stringr boundary
 #' @importFrom parallel clusterApply
 #' @importFrom Rdpack reprompt
+#' @importFrom rlang %||%
```

Comment:
`%||%` can be replaced by something like
```{r}
my_fun <- function(a,b) {
if (is.null(a)) b
else a
}
```

but here we can reduce the code/maintanence with rlang